contracts/interchain-token-service/InterchainTokenService.sol

```diff
@@ -678,11 +676,11 @@ contract InterchainTokenService is
         address tokenAddress = getStandardizedTokenAddress(tokenId);
         address tokenManagerAddress = getTokenManagerAddress(tokenId);
         address distributor = distributorBytes.length > 0 ? distributorBytes.toAddress() : tokenManagerAddress;
+        if (distributor == address(0)) revert ZeroAddress();
         _deployStandardizedToken(tokenId, distributor, name, symbol, decimals, 0, distributor);
-        TokenManagerType tokenManagerType = distributor == tokenManagerAddress ? TokenManagerType.MINT_BURN : TokenManagerType.LOCK_UNLOCK;
         _deployTokenManager(
             tokenId,
-            tokenManagerType,
+            TokenManagerType.MINT_BURN,
             abi.encode(operatorBytes.length == 0 ? address(this).toBytes() : operatorBytes, tokenAddress)
```

Comment:
can you remove ternary operator usage everywhere? Slither can't analyze contracts relying on that.

Reply:
done